tools/wavetool.py

```diff
@@ -112,6 +115,8 @@ def do_wave_analysis():
         analyze_wav_smart_amp(wave, fs_wav)
     if cmd.analyze == 'wov':
         analyze_wav_wov(wave, fs_wav)
+    if cmd.analyze == 'thdn':
+        analyze_wav_thdn(wave, fs_wav)
```

Comment:
Could you add DC offset calculation? Take average value of the clip to analyse and report as dBFS (repeat for every channel). You can omit the start transient. The fail criteria default should be > -60 dBFS. With 16 bit audio it should be normally around -90 dBFS.